### PR TITLE
Layout setup

### PR DESCRIPTION
**Pull request by Zaeemahmed**

### CODE REVIEW

File: pages/board/index.tsx [hunk 1]
```diff
@@ -1,11 +1,24 @@
 import { useQuery } from '@apollo/client';
+import { Row } from 'react-bootstrap';
 import tasksQuery from './tasks';
 
 const Board = () => {
   const { data, loading } = useQuery(tasksQuery, {
```
Comment (Skrpk) [inline pages/board/index.tsx:6]: Should we use the `loading` variable to show some Loading component or we don't care about it for now?

Comment (Zaeemahmed) [reply pages/board/index.tsx:6]: Yes it will be used in the board section component which i am currently working on

File: pages/board/index.tsx [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
   });
-  return <div>Board</div>;
+  return (
+    <div className="pt-3 h-100 d-flex flex-column">
+      <Row>
+        <h1>Project title</h1>
+      </Row>
+      <div className="board-container d-flex flex-row flex-grow-1">
+        {data &&
+          data.tasks.map((task: any) => {
```
Comment (Skrpk) [inline pages/board/index.tsx:16]: Shouldn't we define type for task and use it here?

Comment (Zaeemahmed) [reply pages/board/index.tsx:16]: Yeah you are right, I will create a task component.

**Pull request merged**